pkg/vminetworkscontroller/vmi_controller_test.go

```diff
@@ -330,6 +330,15 @@ func dummyVMISpec(nadName string) virtv1.VirtualMachineInstanceSpec {
 				Name:          "podnet",
 				NetworkSource: virtv1.NetworkSource{Pod: &virtv1.PodNetwork{}},
 			},
+			{
+				Name: "default_multus",
+				NetworkSource: virtv1.NetworkSource{
+					Multus: &virtv1.MultusNetwork{
+						NetworkName: nadName,
+						Default:     true,
+					},
+				},
+			},
```

Comment:
this is a wrong VMI specification.

I don't understand what it means from the kubevirt perspective, but, the right configuration is to omit the pod network.

Please add another method `dummyVMISpecWithMultusDefaultNetwork` and use this one in a new test.

Reply:
i know it is wrong i raised it here https://github.com/kubevirt/ipam-extensions/pull/46#issuecomment-2222586167 fwiw, wondered if it is fine to use it for this sake of testing

anyhow will do as you suggested